src/bin/cargo-flamegraph.rs

```diff
@@ -195,6 +195,26 @@ fn build(opt: &Opt) {
 
         let mut has_debuginfo = false;
 
+        let profile = if opt.bench.is_some() {
+            "bench"
+        } else {
+            "release"
+        };
```

Comment:
Moving this code seems unrelated? Would you mind putting it in a separate commit (explaining why this good)?